backend/src/game/routes.ts

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+import {GameDataModel} from './model';
+import {validateGame} from '../../../frontend/src/app/services/validate';
+
+
+export const router = express.Router();
+
+// router.get('/api/todo', async (req, res) => {
```

Comment:
Consider add TODO or remove it completely.

Reply:
It's a clear todo I will work on today.